packages/xo-web/src/common/zstd-checker.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 
 import _ from './intl'
 import Component from './base-component'
-import constructQueryString from './construct-query-string'
+import constructQueryString from './smartModeToComplexMatcher'
```

Comment:
No default export for `smartModeToComplexMatcher`.

Reply:
Good catch! :pray:

Reply:
Should be fixed but I don't know how to test this component.